src/components/pages/blog-post/blog-post.tsx

```diff
@@ -28,6 +29,15 @@ const PanelContainer = styled.div`
   }
 `;
 
+const RichTextContainer = styled.div`
+  > * {
```

Comment:
Please do it in the styled-components way:

```
const IntroRichText = styled(ContentfulRichText)`
   ....
`
```

Also, `> * {` seems like a too general selector that would also overwrite possible internals in the rich text component. Try to make the selector a bit more strict.

Reply:
I tried to do this, but since the `ContentfulRichText` Component returns an empty JSX.Element tag (which we need for the fullWidth layout in the blog), it did not work out.

This was the only way I was able to do this.